.circleci/config.yml

```diff
@@ -256,26 +317,40 @@ workflows:
       - release-nnoir:
           context:
             - docker-hub-creds
-          filters:
+          filters: &release-filter
             tags:
               only: /^nnoir-[0-9]+(\.[0-9]+){2}((a|b|rc)[0-9]+)?(.post[0-9])?(.dev[0-9])?$/
             branches:
               ignore: /.*/
           dir: nnoir
           package-name: nnoir
+      - build-dockerimage:
+          release: true
+          context:
+            - docker-hub-creds
+          requires:
+            - release-nnoir
+          filters: *release-filter
 
   release-nnoir-onnx:
     jobs:
       - release-nnoir-onnx:
           context:
             - docker-hub-creds
-          filters:
+          filters: &release-filter
```

Comment:
may be `nnoir-onnx-release-filter` is better?

Reply:
fixed